index.html

```diff
@@ -200,11 +200,13 @@ <h2 class="contact-me">Contact Me</h2>
       feature that you need built or a <br> project that needs coding. I’d <br>
       love to help with it </p>
 
-    <input class="name-area" type="text" placeholder="Full name"><br>
-    <input class="mail-area" type="text" placeholder="Email adress"><br>
-    <textarea name="User_message" id="msg" placeholder="Write your message here" cols="30" rows="10"></textarea><br>
-    <button class="end-button" type="submit" name="submit">Get It Touch</button>
-
+    <form action="https://formspree.io/f/moqypzvo" method="post">
+      <input class="name-area" type="text" placeholder="Enter Full name" maxlength="30" required><br>
+      <input class="mail-area" type="email" placeholder="Email adress" required><br>
+      <textarea name="User_message" id="msg" placeholder="Please Write your message here" cols="30" rows="10" maxlength="500"
+        required></textarea><br>
+      <button class="end-button" type="submit" name="submit">Get It Touch</button>
+    </form>
 
```

Comment:
**Changes Required**
Well done implementing the form validation, use of placeholders, and providing `maxlength`. However;
- [ ] The form layouts design should be close to the figma design as much as possible for both mobile and web. Converting UI/UX design into perfect code is one of the sought for skills of front-end developers.
Please check the difference below.

**Your Desktop Form**
![your-desktop-form](https://user-images.githubusercontent.com/7220167/130703579-2e55887e-0369-48bd-a81f-425f3ccd6a67.png)
**Provided Figma Desktop Form**
![desktop-form](https://user-images.githubusercontent.com/7220167/130703611-39faaa1d-4a64-4231-9539-21b28cdadf5b.png)
**Your Mobile Form**
![your-mobile-form](https://user-images.githubusercontent.com/7220167/130703641-6cd39d4b-a38d-4104-9220-4fa8c3d1ce8f.png)
**Provided Figma Form**
![mobile-form](https://user-images.githubusercontent.com/7220167/130703679-e44b37e0-c2a6-4e35-9f0b-d9b53411d190.png)

Reply:
Thank you for your constructive reviews but the project here is just a mobile version, we didn't get to the desktop version yet.